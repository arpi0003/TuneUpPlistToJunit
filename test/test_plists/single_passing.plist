<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>All Samples</key>
	<array>
		<dict>
			<key>LogType</key>
			<string>Pass</string>
			<key>Message</key>
			<string>Holding login button shows the edit url popup</string>
			<key>Timestamp</key>
			<date>2014-03-04T20:36:27Z</date>
			<key>Type</key>
			<integer>4</integer>
		</dict>
		<dict>
			<key>LogType</key>
			<string>Pass</string>
			<key>Message</key>
			<string>Holding login button shows the edit url popup</string>
			<key>Timestamp</key>
			<date>2014-03-04T20:36:41Z</date>
			<key>Type</key>
			<integer>5</integer>
		</dict>
		<dict>
			<key>LogType</key>
			<string>Debug</string>
			<key>Message</key>
			<string>target.frontMostApp().mainWindow().tableViews()["Menu Table"].cells()["Log Out"].tap()</string>
			<key>Timestamp</key>
			<date>2014-03-04T20:39:11Z</date>
			<key>Type</key>
			<integer>0</integer>
		</dict>
		<dict>
			<key>LogType</key>
			<string>Pass</string>
			<key>Message</key>
			<string>Clicking the log out button starts the log out process</string>
			<key>Timestamp</key>
			<date>2014-03-04T20:39:13Z</date>
			<key>Type</key>
			<integer>5</integer>
		</dict>
	</array>
	<key>version</key>
	<integer>2</integer>
</dict>
</plist>
