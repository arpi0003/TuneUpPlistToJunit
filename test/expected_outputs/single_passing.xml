<?xml version="1.0"?>
<testsuite tests="1" errors="0" failures="0" timestamp="2014-03-04T20:36:27+00:00" time="14.0">
  <testcase name="Holding login button shows the edit url popup" time="14.0"/>
</testsuite>
